tutorials/information_theoretic_acquisition_functions.ipynb

```diff
@@ -264,16 +267,8 @@
     "from botorch.fit import fit_gpytorch_mll\n",
     "\n",
     "tkwargs = {\"dtype\": torch.double, \"device\": \"cpu\"}\n",
-    "SMOKE_TEST = os.environ.get(\"SMOKE_TEST\")"
```

Comment:
You need to define that var; looks like the tutorial fails since you removed this.